DPG/Tasks/AOTTrack/tagAndProbeDmesons.cxx

```diff
@@ -390,27 +552,32 @@ struct TagTwoProngDisplacedVertices {
           continue;
         }
 
-        auto pVec = RecoDecay::pVec(pVecTrackFirst, pVecTrackSecond);
         auto primVtx = getPrimaryVertex(collision);
         const auto& secVtx = vertexer.getPCACandidate();
         const auto& covMatrixPCA = vertexer.calcPCACovMatrixFlat();
         std::array<float, 2> trackDcaXy{trackFirst.dcaXY(), trackSecond.dcaXY()};
-        if (!isSelectedTopology(primVtx, secVtx, covMatrixPCA, pVec, trackDcaXy, channel)) {
+        if (fillTagTable) {
+          getTagInfo<doMc>(primVtx, secVtx, covMatrixPCA, pVec, trackDcaXy, channel, trackFirst, trackSecond, invMass2, pdgDecayMothers, particlesMc);
           continue;
```

Comment:
do you really need it? Aren't you at the end of the iteration?

Reply:
Same as above.